presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/FunctionCallProvider.java

```diff
@@ -62,6 +62,40 @@ public FunctionCall getExpectedValue(SymbolAliases aliases)
         return new FunctionCall(name, symbolReferences);
     }
 
+    public static ExpectedValueProvider<FunctionCall> windowFunctionCall(String name, List<String> args)
+    {
+        return new FunctionCallProvider(true, QualifiedName.of(name), false, toSymbolAliases(args));
+    }
+
+    public static ExpectedValueProvider<FunctionCall> functionCall(
+            String name,
+            boolean distinct,
+            List<PlanTestSymbol> args)
+    {
+        return new FunctionCallProvider(false, QualifiedName.of(name), distinct, args);
+    }
+
+    public static ExpectedValueProvider<FunctionCall> functionCall(String name, List<String> args)
+    {
+        return new FunctionCallProvider(QualifiedName.of(name), toSymbolAliases(args));
+    }
+
+    private static List<PlanTestSymbol> toSymbolAliases(List<String> aliases)
+    {
+        return aliases
+                .stream()
+                .map(PlanMatchPattern::symbol)
+                .collect(toImmutableList());
+    }
+
+    public static List<Expression> toSymbolReferences(List<PlanTestSymbol> aliases, SymbolAliases symbolAliases)
```

Comment:
Even though the only caller is in this class, I'd probably leave this in PMP to make it more discoverable for the next person messing with the code. Up to you.

This code is at the point where many people are hacking on it slowly over time, not one person who has all of the code in their head iterating fast.